### DISCUSSION TITLE
Customizeable Login Screen

### DISCUSSION THREAD

**Discussion by LaCocoRoco** [coder/code-server#5569] [Ideas] [answered]
To customize the login screen i changed files in the "code-server/src/browser/pages" folder.
On every update i have to do this again. So i thought to request the feature for a integrated solution.

![image](https://user-images.githubusercontent.com/10038346/190918341-affee594-43fd-4a28-bdbb-fe1fd9c0c57e.png)

**Comment by jsjoeio** [comment, marked as answer]
This might be related: https://github.com/coder/code-server/issues/3019

Feel free to add a comment there!
